docs/docs/dev/debug/index.md

```diff
@@ -0,0 +1,5 @@
+The xemu source tree can be quite intimidating at first due to its qemu roots, but the majority of Xbox-specific code is confined to the `xemu/hw/xbox/` folder where the rest can be disregarded.

```

Comment:
* nit: QEMU is stylized in all uppercase (in contrast with xemu which is stylized in all lowercase). Let's change mentions of QEMU in this PR to use the uppercase convention.
* Let's drop the `due to its qemu roots`
* Let's change `, but the majority of Xbox-specific code` to `. Xbox-specific hardware emulation code`
* Let's drop the `rest can be disregarded` line. It depends on what someone is fixing, and many things do live outside of hw/xbox, like all of the UI, core networking stuff, main loop, etc.